sos/report/plugins/kubernetes.py

```diff
@@ -83,21 +113,9 @@ def setup(self):
             'KUBERNETES_NO_PROXY'
         ])
 
-        svcs = [
-            'kubelet',
-            'kube-apiserver',
-            'kube-proxy',
-            'kube-scheduler',
-            'kube-controller-manager',
-            'snap.kubelet.daemon',
-            'snap.kube-apiserver.daemon',
-            'snap.kube-proxy.daemon',
-            'snap.kube-scheduler.daemon',
-            'snap.kube-controller-manager.daemon'
-        ]
-
-        for svc in svcs:
-            self.add_journal(units=svc)
+        for svc in self.services:
+            if self.is_service_enabled(svc):
```

Comment:
This is a change from before. Now if the service is not enabled, or for some reason someone has accidentally disabled the service, but we may still have journal logs, I think we should not have this check.

Reply:
would this be ok ? (that's what I really wanted actually) It's a bit confusing to have empty log files for services that just don't exists
```suggestion
            if self.is_service(svc):
```

Reply:
I know it's big **if**, what if the package is removed, and we still have the journals there to consume? 🤔

Reply:
The plugin might not even start to collect if enough stuff are deleted, and recent logs will be collected by the logs plugin.
I'll follow whatever you prefer, but this is a weird corner case to account for IMO